Code/Legacy/CrySystem/Log.cpp

```diff
@@ -821,13 +810,13 @@ void CLog::PushAssetScopeName(const char* sAssetType, const char* sName)
     SAssetScopeInfo as;
     as.sType = sAssetType;
     as.sName = sName;
-    CryAutoCriticalSection scope_lock(m_assetScopeQueueLock);
+    AZStd::lock_guard<AZStd::mutex> scope_lock(m_assetScopeQueueLock);
```

Comment:
This can be changed to be

```suggestion
    AZStd::scoped_lock scope_lock(m_assetScopeQueueLock);
```